mne_bids/write.py

```diff
@@ -1070,6 +1084,52 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
              'NAS': list(mri_landmarks[1, :]),
              'RPA': list(mri_landmarks[2, :])}
         fname = t1w_basename.replace('.nii.gz', '.json')
+        if op.isfile(fname) and not overwrite:
+            raise IOError('Wanted to write a file but it already exists and '
+                          '`overwrite` is set to False. File: "{}"'
+                          .format(fname))
         _write_json(fname, t1w_json, overwrite, verbose)
 
+        if deface:
+            # x: L/R L+, y: S/I I+, z: A/P A+
+            t1w_data = t1w.get_data().copy()
+            indices = np.meshgrid(np.arange(t1w_data.shape[0]),
+                                  np.arange(t1w_data.shape[1]),
+                                  np.arange(t1w_data.shape[2]),
+                                  indexing='ij')
+            indices = np.array(indices)  # e.g., (3, 86, 86, 86)
+            indices = np.transpose(indices, [1, 2, 3, 0])  # (86, 86, 86, 3)
+            indices = indices.reshape(-1, 3)  # (86 * 86 * 86, 3)
+
+            mri_landmarks2 = apply_trans(t1w.affine, mri_landmarks)
+            meg_trans = get_ras_to_neuromag_trans(*mri_landmarks2[[1, 0, 2]])
+
+            indices = apply_trans(t1w.affine, indices)
+            indices = apply_trans(meg_trans, indices)
+            trans_y = -mri_landmarks2[1, 1] + t1w_data.shape[2] * inset
+            indices = apply_trans(translation(y=trans_y), indices)
+            indices = apply_trans(rotation(x=-np.deg2rad(theta)), indices)
+            coords = indices.reshape(t1w.shape + (3,))
+            mask = (coords[..., 2] < 0)
+
+            t1w_data[mask] = 0.
+            # smooth decided against for potential lack of anonymizaton
+            # https://gist.github.com/alexrockhill/15043928b716a432db3a84a050b241ae
+
+            t1w = nib.Nifti1Image(t1w_data, t1w.affine, t1w.header)
+            if plot_on:
+                fig = t1w.orthoview()
+                fig.show()
```

Comment:
you'll soon have people asking to modify the plots, let's not do that. Modular is better.